components/AboutCairoVMBanner.tsx

```diff
@@ -23,7 +27,11 @@ const AboutCairoVMBanner = () => {
   }
 
   return (
-    <div className="relative bg-gray-50 dark:bg-black-700 pb-6 mt-0 mb-10">
+    <div
```

Comment:
Better not to render the component at all in case its full screen, instead of using `hidden` class.

See the "Inline If with Logical && Operator" section here: https://legacy.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

So it will look like:
``` 
!isFullscreen && (
  <div className ... ></div>
)
```